index.txt

```diff
@@ -39,16 +39,16 @@ install_caddy()
 
 	# Termux on Android has $PREFIX set which already ends with /usr
 	if [[ -n "$ANDROID_ROOT" && -n "$PREFIX" ]]; then
-		install_path="${PREFIX}/bin"
+		install_path="$PREFIX/bin"
```

Comment:
The brace syntax is sometimes mandated, like when addressing array elements, or in case a character follows a variable name that could be part of a variable. But when followed by slash, dot, and some such, the braces are of zero benefit, and hinder readability.